numberninecms/cms/0.5/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "bundles": {
+        "NumberNine\\NumberNineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "copy-from-package": {
+        "install/docker/": "docker/",
+        "install/public/": "%PUBLIC_DIR%/"
+    },
```

Comment:
Hey @williarin, I'm a bit confused about these instructions. I looked in the package and from the `docker` directory it copies some config files for NGINX and PHP, but those are useless without the Docker Compose configuration.
What's the idea behind it?

Reply:
Hi, there's a docker-compose.yml in the bundle repo but it cannot be used in the recipe as it would be incompatible with the one Symfony creates. The yml is either copied by the installer (separate repo) or by a console command I didn't make yet.

Reply:
You could use the `docker-compose` configurator. Here's an example: https://github.com/symfony/recipes/blob/296e45d39a760ff9aca2fb9ffa4f6fa6a4b9bb99/doctrine/doctrine-bundle/2.4/manifest.json#L23-L39

Reply:
Can the configurator delete a service created by another recipe? I didn't use this method because I need doctrine-bundle which creates this Postgre service, but also need MySQL and not Postgre.

Reply:
No, it cannot delete, but AFAIK it works per service, if you define another `database` service as Doctrine does, then it will update its config.

Reply:
And to maintain the compatibility, you can add a `mysql` alias to the `database` service.

Reply:
I'll try that, thank you.

Reply:
I updated the recipe with the docker-compose configurator. It creates duplicate services and volumes instead of replacing them, but it's not a big deal after all as the installer takes care of cleaning the file.